Dangerfile.swift

```diff
@@ -66,18 +66,17 @@ if pr.assignees?.count == 0 {
     warn("Please assign someone aside from CODEOWNERS (@checkout-pci-reviewers) to review this PR.")
 }
 
-
 // MARK: - SwiftLint
 
 // Use a different path for SwiftLint
 
-//let files = sdkEditedFiles.filter { $0.fileType == .swift }
-//SwiftLint.lint(.files(files), inline: true, swiftlintPath: "Sources/.swiftlint.yml")
+// let files = sdkEditedFiles.filter { $0.fileType == .swift }
+// SwiftLint.lint(.files(files), inline: true, swiftlintPath: "Sources/.swiftlint.yml")
```

Comment:
What do you think about having Danger run the linter on changed files? I think it would tighten the feedback loop and give us more actionable errors than a failing build later in the pipeline. cc @jnewc

Reply:
Agree 💯

Reply:
That is a great idea! That can be added as a step after we fix all the warnings.

Reply:
Cool. We can probably do it sooner than that as it will be working off the incremental rule set, but we can do it in the next round of updates.